R/cache.R

```diff
@@ -268,7 +272,7 @@ cache <- function(uncached_function, key, salt, con, prefix = deparse(uncached_f
       ),
       parent = environment(uncached_function))
 
-  build_cached_function(cached_function, safe_columns)
+  build_cached_function(cached_function, safe_columns, blacklist)
```

Comment:
This is what I mean. We should not be haphazardly adding on parameters like this. You can store this information in the function call metadata.